src/NuGet.Core/NuGet.ProjectModel/PackageSpecWriter.cs

```diff
@@ -210,6 +214,7 @@ private static void WriteMetadataBooleans(IObjectWriter writer, ProjectRestoreMe
             SetValueIfTrue(writer, "centralPackageFloatingVersionsEnabled", msbuildMetadata.CentralPackageFloatingVersionsEnabled);
             SetValueIfTrue(writer, "centralPackageVersionOverrideDisabled", msbuildMetadata.CentralPackageVersionOverrideDisabled);
             SetValueIfTrue(writer, "CentralPackageTransitivePinningEnabled", msbuildMetadata.CentralPackageTransitivePinningEnabled);
+            SetValueIfTrue(writer, "UsingMicrosoftNETSdk", msbuildMetadata.UsingMicrosoftNETSdk);
```

Comment:
From my interpretation of https://github.com/NuGet/NuGet.Client/pull/5833#discussion_r1672759592
it was not necessary to change this to write only when true.

Reply:
Andy's right. 
We werent' asking for this to be changed.